README.md

```diff
@@ -4,21 +4,17 @@ Open source is made by people just like you. This Friday, invest a few hours con
 
 ## What is this?
 
-Open Source Friday is a movement to encourage companies, people and maintainers to contribute a few hours contributing to open source every Friday.
-
-*Alternative name: 52 pull requests*
+This repository is the source code for https://ossfriday.com
```

Comment:
Yes! I think this makes it clear now that this the app repository and not the Open Source Friday guidelines or tutorials.